src/components/QuestDetailsWrapper/index.tsx

```diff
@@ -107,7 +109,9 @@ export function QuestDetailsWrapper({
   tOverride,
   sessionEmail,
   checkG7ConnectionStatus,
-  isQuestsPage
+  isQuestsPage,
+  syncPlayStreak,
+  ...questDetailsParamProps
```

Comment:
feels like we are going against the existing pattern here by adding the spread operator just for `i18n` instead of just specifying `i18n`. I think it is making the code a bit more difficult to read/reason about quickly given the existing pattern